tools/assist.py

```diff
@@ -171,6 +175,18 @@ def do_query(query):
         i += 1
     print()
 
+
+def do_cross_query():
+    clusters = []
+
+    for sym_name in map_syms:
+        sym = map_syms[sym_name]
+        if get_symbol_length(sym_name) > 8:
+            cluster_match = False
+            for cluster in clusters:
+                pass # todo do
```

Comment:
Left a todo here, is it meant to be done?

Reply:
yeah, I'll do it over time but it's just an enhancement

Reply:
IMO unfinished stuff that doesn't somewhat have use (since this seems like it won't output anything?) shouldn't be merged in, but thats your call

Reply:
I generally agree, but since I'm the only one who uses this tool, I don't think this affects anyone, similar to how substitute.py exists and I'm the only one who uses that. But I'll revert it for now